### DISCUSSION TITLE
[turbo-ux] asset mapper path problem, could not find turbo-core. Symfony 7.1.8

### DISCUSSION THREAD

**Discussion by thomasahaf** [symfony/symfony#58909] [Q&A] [answered]
Hi, I just started getting problems with symfony/ux-turbo after starting a new Symfony 7.1.8 project. I start with the command `symfony new my_project_directory --version="7.1.*" --webapp`  

Creating a new controller and a twig template that extends the base.html.twig, gives the error. `An exception has been thrown during the rendering of a template ("Could not find an asset mapper path that points to the "turbo-core" controller in package "symfony/ux-turbo", defined in controllers.json.").` 

Also this command gives the same error `php bin\console asset-map:compile` 

This happens on a newly created project, on a project created before and upgraded to 7.1.8 it works. 

I am not sure what I am doing wrong here or if something happened to the asset-mapper in 7.1.8

If anyone can guide me I would be grateful.

**Comment by joerleigh** [comment]
I had this same issue this morning. Check to see if TurboBundle is included in bundles.php. I just installed the bundle in a project this morning and the composer recipe did not run. It looks like the latest ux-turbo composer recipe will not run on Symfony versions <7.2, but Symfony 7.2 is not yet released.

**Comment by sharovik** [comment, marked as answer]
As temp solution, you can add TurboBundle class in `bundles.php` manually.
```php
<?php

return [
    //...
    Symfony\UX\Turbo\TurboBundle::class => ['all' => true],
    //...
];
```

**Comment by azy64** [comment]
Hi everyone!
@thomasahaf , you need to include this:  Symfony\UX\Turbo\TurboBundle::class => ['all' => true], in the bundles PHP file

**Comment by thomasahaf** [comment]
Thank you all, that fixed it. Adding TurboBundle::class to the bundle.php file fixed it.
